source/protobuf/session.proto

```diff
@@ -54,6 +58,18 @@ message EnumerateDevicesResponse {
   repeated DeviceProperties devices = 1;
 }
 
+message SoftwareProperties {
+  string package_id = 1;
+  string version = 2;
```

Comment:
To be more future-proof, I'd suggest calling this `package_version`. The reason is that we're likely to add "display version" in the near future, you know, the 22Q4 kind of stuff. This field needs to clearly be the dotted version that you can pass to normal version comparison routines.